### PR TITLE
Added save entity command class generation

### PR DESCRIPTION
**Pull request by bohdan-harniuk**
**Description** (*)
Added save entity command service generation and covered with the test case.

Screenshot 2021-01-27 at 00 01 02
Screenshot 2021-01-27 at 00 12 20
Screenshot 2021-01-27 at 00 11 51

**Contribution checklist** (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with integration/functional tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: src/com/magento/idea/magento2plugin/magento/files/commands/SaveEntityCommandFile.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    public static final String TEMPLATE = "Magento Save Entity Command Model";
+    private static final String DIRECTORY = "Command";
+    private static final SaveEntityCommandFile INSTANCE = new SaveEntityCommandFile();
+
+    /**
+     * Get singleton object of save command file.
+     *
+     * @return SaveEntityCommandFile
+     */
+    public static SaveEntityCommandFile getInstance() {
```
Comment (VitaliyBoyko) [inline src/com/magento/idea/magento2plugin/magento/files/commands/SaveEntityCommandFile.java:27]: No singletons. And this is not actually  a singleton 🤣

Comment (VitaliyBoyko) [reply src/com/magento/idea/magento2plugin/magento/files/commands/SaveEntityCommandFile.java:27]: Lets use just a constructor.

File: src/com/magento/idea/magento2plugin/magento/packages/code/PhpCoreType.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+package com.magento.idea.magento2plugin.magento.packages.code;
+
+import org.jetbrains.annotations.NotNull;
+
+public enum PhpCoreType {
+    EXCEPTION("Exception");
```
Comment (VitaliyBoyko) [inline src/com/magento/idea/magento2plugin/magento/packages/code/PhpCoreType.java:11]: Entire Enum for the only one constant?

**Review by VitaliyBoyko — CHANGES_REQUESTED** [review]

File: src/com/magento/idea/magento2plugin/magento/files/commands/SaveEntityCommandFile.java (hunk 1 above)
Comment (bohdan-harniuk) [reply src/com/magento/idea/magento2plugin/magento/files/commands/SaveEntityCommandFile.java:27]: Fixed, thank you!)

File: src/com/magento/idea/magento2plugin/magento/packages/code/PhpCoreType.java (hunk 2 above)
Comment (bohdan-harniuk) [reply src/com/magento/idea/magento2plugin/magento/packages/code/PhpCoreType.java:11]: Removed, thank you!

**Review by VitaliyBoyko — APPROVED** [review]

**Pull request merged**